### ISSUE TITLE
UserDefinedGravityAcceleration redeclared with modifier mue

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1361]
**Reported by petar on 22 Nov 2013 12:42 UTC**
The model UserDefinedGravityField in MultiBody redeclares the gravityAcceleration function with the following modifiers:

```
redeclare function gravityAcceleration = 
Modelica.Mechanics.MultiBody.Examples.Elementary.Utilities.theoreticalNormalGravityWGS84
(mue = 1, phi = geodeticLatitude), axisLength = 10, nominalLength = 10) 
```
But there is no mue component in theoreticalNormalGravityWGS84. There is one in the World object, but that is probably not legal to access (I hope).

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1361

**Comment by modelica-trac-importer** [comment]
**Modified by beutlich on 21 May 2014 19:03 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 20 Jun 2014 13:51 UTC**
Fixed in 3f3a5d60f483076139938f5fc008d0f6d8d91ece (trunk) and 1b0f047b6a0dbe7045afda383680ab9d4e5746a8 (maintenance)
